src/components/30-organisms/table-sortable/ui.test.js

```diff
@@ -0,0 +1,217 @@
+import { Selector } from 'testcafe';
```

Comment:
Why do you use $ for variable names everywhere in here? Just curious, since we don't do that in other tests.

I know for instance from angularjs that everything prefixed with $ is a framework related variable, but I don't see the connection to here. $router in angular for instance, is the angular router you can inject everywhere, but here, the $ prefixed variables are not injected, they were declared by us.

Reply:
marco did and i guess is a best practices that comes from jQuery where $ represents a living DOM node